### PR TITLE
feat: workflow unpublished entry

### LINKED ISSUE #727: Unpublishing for the editorial workflow
**- Do you want to request a *feature* or report a *bug*?**

feature

**- What is the current behavior?**

Published posts cannot be unpublished.

**- What is the expected behavior?**

User should be able to unpublish a published post:

- Push a commit to the published branch to remove the entry
- Create a new branch with an additional commit to re-add the removed entry
- PR the new branch
- Set status in the editorial workflow to Ready to Publish

### PR DESCRIPTION
**Pull request by barthc**
**Summary**
Closes #727 

Currently swap the editorial toolbar section from:
unpublish-before
 
to:
unpublish-after

cc: @tomrutgers

### CODE REVIEW

**Comment by tomrutgers** [comment]
I'm not entirely sure about the color scheme for the button, I associate green with actions like publishing, not unpublishing. Maybe we should use a gray version. 

I prefer this one
button-gray

Over the default:
button-gray-default

The other colored buttons used are a bit harsh IMHO.
colored-buttons

Thoughts?

**Comment by barthc** [comment]
If we are going gray for  `unpublish` , then we also need to modify the color scheme for buttons  `save`  ,`Delete published entry`, etc  just to be consistent.

**Comment by tomrutgers** [comment]
Why? It's already a different color button..

File: packages/netlify-cms-locales/src/en/index.js [hunk 1]
```diff
@@ -63,6 +64,8 @@ const en = {
       publishing: 'Publishing...',
       publish: 'Publish',
       published: 'Published',
+      unpublish: 'Unpublish entry',
+      unpublishing: 'Unpublishing',
```
Comment (erezrokah) [inline packages/netlify-cms-locales/src/en/index.js]: Should be `Unpublishing...` similar to `Publishing...` right?

**Comment by erezrokah** [comment]
How about:

image

Or:

image

File: packages/netlify-cms-locales/src/en/index.js [hunk 2]
```diff
@@ -63,6 +64,8 @@ const en = {
       publishing: 'Publishing...',
       publish: 'Publish',
       published: 'Published',
+      unpublish: 'Unpublish entry',
```
Comment (erezrokah) [inline packages/netlify-cms-locales/src/en/index.js]: Can we use just `Unpublish` like `Publish and create new` or `Publish now`?

**Comment by erezrokah** [comment]
@barthc looks good pending UI decisions. Also do you mind adding a scenario for unpublishing for the test backend in the Cypress tests?

**Comment by tomrutgers** [comment]
@erezrokah Digging that first one!

**Comment by erezrokah** [comment]
> @erezrokah Digging that first one!

Just thinking we should be consistent with one of these:
image

**Comment by tomrutgers** [comment]
Even better!

**Comment by barthc** [comment]
@erezrokah  honestly the dropdown `unpublish` was my first approach but later changed it  :grinning:

**Comment by tomrutgers** [comment]
That settles it then 😄

**Comment by erezrokah** [comment]
@tomrutgers, @barthc  I rebased the branch, added a cypress test and a fix to re-load the unpublished entry (was needed due to this recent change: https://github.com/netlify/netlify-cms/pull/2878/files#diff-7d0283827744acf7472c54d35c205500R305).

**Comment by barthc** [comment]
@erezrokah  was about to push changes for the dropdown and the cypress test  :grinning:

**Comment by tomrutgers** [comment]
Looking good!

**Comment by erezrokah** [comment]
@barthc Sorry for that, I'm pushing for keeping the PR number low and we have some more changes coming around that are :)

**Pull request merged**